.github/workflows/publish-development.yaml

```diff
@@ -57,11 +57,11 @@ jobs:
         target: ${{ steps.setname.outputs.build }}
 
     - name: Crosslink flist (development)
-      if: success() && github.ref == 'refs/heads/master'
+      if: success()
```

Comment:
you should have only change it to `refs/heads/master-rework`.

Reply:
Tried that, doesnt work

Reply:
Will work once merged, since now the branch ref is `refs/heads/fix_publish_flist`